website/docs/concepts/reading.mdx

```diff
@@ -74,35 +74,41 @@ A typical [ConsumerWidget] would look like:
 Similar to [ConsumerWidget], [ConsumerStatefulWidget] and [ConsumerState] are the equivalent of a
 [StatefulWidget] with its [State], with the difference that the state has a "ref" object.
 
-This time, the "ref" isn't passed as parameter of the build method, but is rather
+This time, the "ref" isn't passed as parameter of the build method, but is
 a property of the [ConsumerState] object:
 
 <CodeBlock>{trimSnippet(consumerStatefulWidget)}</CodeBlock>
 
 #### Extending HookConsumerWidget instead of HookWidget
 
-This solution is specific to [flutter_hooks] users. Since [flutter_hooks] requires
+This option is for [flutter_hooks] users. Since [flutter_hooks] requires
 extending [HookWidget] to work, widgets that use hooks are unable to extend
 [ConsumerWidget].
 
-One solution is, through the package [hooks_riverpod], replace [HookWidget]
-by [HookConsumerWidget].  
+The package [hooks_riverpod] exposes a new widget called [HookConsumerWidget].
 [HookConsumerWidget] acts as both a [ConsumerWidget] and a [HookWidget]. This
 allows a widget to both listen to providers and use hooks.
 
 An example would be:
 
 <CodeBlock>{trimSnippet(consumerHookWidget)}</CodeBlock>
 
+#### Extending StatefulHookConsumerWidget instead of HookWidget
+
+This option is for [flutter_hooks] users. Although Hooks and riverpod [Provider]s are a general replacement for
+management of local state, there has been requests for a StatefulWidget version of [HookConsumerWidget].
```

Comment:
I don't think saying "there has been requests for it" makes sense in the documentation.